gcc-11.3.0/gcc/toplev.c

```diff
@@ -2049,9 +2060,14 @@ finalize (bool no_backend)
 
   /* Close non-debugging input and output files.  Take special care to note
      whether fclose returns an error, since the pages might still be on the
-     buffer chain while the file is open.  */
-
-  if (asm_out_file)
+     buffer chain while the file is open.  If the calculation of the GitBOM
+     information is enabled, do not close the asm_out_file because the gitoid
+     of the resulting object file is to be written in the .bom section later,
```

Comment:
It should be *gitoid of the resulting gitbom file* is to be written to the .bom section; this piece again will change once we implement the recording of gitoids in the NOTE section